botorch_community/acquisition/latent_information_gain.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+Latent Information Gain Acquisition Function for Neural Process Models.
+
+References:
+
+.. [Wu2023arxiv]
+   Wu, D., Niu, R., Chinazzi, M., Vespignani, A., Ma, Y.-A., & Yu, R. (2023).
+   Deep Bayesian Active Learning for Accelerating Stochastic Simulation.
+   arXiv preprint arXiv:2106.02770. Retrieved from https://arxiv.org/abs/2106.02770
+
+Contributor: eibarolle
+"""
+
+from __future__ import annotations
+
+import warnings
+from typing import Optional
+
+import torch
+from botorch import settings
+from botorch_community.models.np_regression import NeuralProcessModel
+from torch import Tensor
+
+import torch
+#reference: https://arxiv.org/abs/2106.02770 
+
+class LatentInformationGain:

```

Comment:
Can we implement this as a subclass of `Acquisition`, so that we can use it more organically in botorch? Likely the context would be needed to be provided in `LatentInformationGain.__init__`